stdlib/public/TensorFlow/Utilities.swift

```diff
@@ -176,3 +176,17 @@ public func _addScalarTensorsWithShape<Scalar: TensorFlowNumeric>(
 ) -> Tensor<Scalar> {
   return Raw.add(x, y)
 }
+
+// TODO: Consider revising the call sites where this is necessary to only need
+// UnsafeMutablePointer to optional when it is the actual c-api call site.
+extension UnsafeMutablePointer where Pointee == CTensorHandle? {
+  @usableFromInline
+  init(_ other: UnsafeMutablePointer<CTensorHandle>) {
```

Comment:
Hmm why is this necessary? Isn’t this just doing a copy?

Reply:
Notice Pointee is Optional, but it is taking from a non-optional Pointee.

Reply:
Oh I see.